src/abort-controller-shim.ts

```diff
@@ -15,10 +18,10 @@ export class AbortSignal {
 		listeners.push(listener);
 	}
 
-	removeEventListener(type, listener) {
+	removeEventListener(type: string, listener: () => void) {
 		const listeners = this._events[type];
 		if (listeners) {
-			const index = listeners.find((x) => x === listener);
+			const index = listeners.findIndex((x: () => void) => x === listener);
```

Comment:
`listeners` here is an array of functions. Calling `findIndex` on it means that the `x` here will be a void returning function. But, because the type of listeners (`_events[type]`) is defined at the top of this class, this type was redundant and has been removed :)